.github/workflows/docker.yml

```diff
@@ -114,7 +117,7 @@ jobs:
       - name: Upload digest
         uses: actions/upload-artifact@v3
         with:
-          name: digests
+          name: digests-${{ steps.meta.outputs.tags }}
```

Comment:
This is sufficient if only the tags differ, but if we want to push to multiple registries by just building the image twice, we still end up with duplicates in here, right? We could also include the image name and registry to circumvent that problem.

Reply:
I think so, yes. If we add registry and image name we probably have everything covered that might come up in the future